config/_shared.json

```diff
@@ -122,6 +122,7 @@
 	"rtl": false,
 	"siftscience_key": false,
 	"signup_url": false,
+	"new_onboarding_url": false,
```

Comment:
If I may nitpick on the config key naming here: `new` can lose its meaning outside of the context of this present project, especially for someone who hasn't worked on it that's looking at the code in half a year from now. Is it possible to use terms that make sense and bring additional context?

Reply:
This seems to be the official name ( pcr1an-2g-p2 ) but I agree that it makes sense to replace it here with the former name of the flow which is mostly used in development (including in the project files [path](https://github.com/Automattic/wp-calypso/tree/trunk/client/landing/gutenboarding)).

Reply:
The "Gutenboarding" name is used very consistently across the codebase and is much more unique and descriptive than just "new". This name is not intended to be ever seen by the user in the UI, but for naming variables or feature flags, it's the right name to use. Here, `gutenboarding_url` is guaranteed to be understood even after many months and years.

Reply:
`gutenboarding_url` makes so much more sense indeed 👍  Thanks for considering it @razvanpapadopol